uhabits-core/src/jvmTest/java/org/isoron/uhabits/core/models/ScoreListTest.kt

```diff
@@ -145,6 +146,93 @@ class YesNoScoreListTest : BaseScoreListTest() {
         checkScoreValues(expectedValues)
     }
 
+    @Test
+    fun test_getNumberOfSkipsByInterval_NoSkips() {
+        val vars = intArrayOf(-1, -1, -1, -1, 3, 2, 2, -1, 2, 2, -1, 2, 2, -1, 3, 2)
```

Comment:
Here and below: can you hide the integer behind the enum, i.e. use `SKIP` etc. instead?